### DISCUSSION TITLE
Execute master branch and specific Git tag on schedule

### DISCUSSION THREAD

**Discussion by cytopia** [community/community#25506] [Actions]
<p>I have a workflow file that does:</p>
<ol>
<li>build docker image</li>
<li>test docker image</li>
<li>push docker image</li>
</ol>
<p>The docker image tag is derived from the git branch or tag:<br>
&lt;IMAGE&gt;:latest (for master branch)<br>
&lt;IMAGE&gt;:&lt;GIT_BRANCH&gt;<br>
&lt;IMAGE&gt;:&lt;GIT_TAG&gt;</p>
<p>Now I want to make sure that the <strong>:latest</strong> and the <strong>last two git tags</strong> are built on a weekly base via GitHub Actions <strong>schedule</strong>.</p>
<p>I know how to define a schedule and limit a job via the condition to only run for <strong>master</strong> branch.</p>
<hr>
<p>**But how to I tell GitHub actions, that it should also run for the last two git tags?</p>
<p>Note:**</p>
<p>This is how it can be done via Travis CI:</p>
<p><strong><img src="//discourse-cloud-file-uploads.s3.dualstack.us-west-2.amazonaws.com/github/original/2X/8/8d9ff64aeca14548667e1babe2a90027c5486e01.png" alt="Screenshot 2019-11-05 14-25-44 [selection].png" width="" height=""></strong></p>

**Comment by eddiechenms** [comment, marked as answer]
<p>Base on your screenshots, seems like you are updating the tags to build manually in Travis CI. I didn’t find any way to set the last two tags automatically, if you can update the last two tags to build in the workflow file manully when there is new tag created, you can create a workflow configuration similar as below:</p>
<pre><code>on:
   schedule:
    - cron: '0 0 * * 1'

jobs:
  build:
    strategy:
      matrix:
        refs: [refs/heads/master, refs/tags/tag1, refs/tags/tag2]

    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v1
      with:
        ref: ${{ matrix.refs }}
    - name: build docker image
    - name: test docker image
    - name: push docket image
</code></pre>
<p>This will run three jobs when the workflow scheduled to run and the jobs will run the build with the source from master branch, tag1 and tag2 respectively. </p>

**Comment by ekohl** [comment]
<p>Will this also run on every fork of the repository? Is there a way to limit the schedule to just a single organization?</p>
